spec/integration/categories_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
```

Comment:
![Screenshot_534](https://user-images.githubusercontent.com/69638013/202437426-c36725de-e63a-486e-a9fb-0568a613cad1.png)

- [ ] Kindly make sure that the tests written are passing
- [ ] Please make sure App has unit and integration tests for all the most important components of the RoR application.
 

- [ ] Please update your readme with instructions to run tests 👍🏿

Reply:
- The tests are passing, the issue might be related to Operating System.

![image](https://user-images.githubusercontent.com/17243325/202479051-a0064bdc-a0a1-4931-8d52-c4a14bef75ef.png)